Jellyfin.Api/Controllers/PlaylistsController.cs

```diff
@@ -371,6 +372,7 @@ public async Task<ActionResult> RemoveUserFromPlaylist(
     public async Task<ActionResult> AddItemToPlaylist(
         [FromRoute, Required] Guid playlistId,
         [FromQuery, ModelBinder(typeof(CommaDelimitedCollectionModelBinder))] Guid[] ids,
+        [FromQuery] bool? moveToTop,
```

Comment:
For the API design it would be better to add a "position" parameter instead of a boolean. That way a newly added item can be placed anywhere in the playlist. With 0 being the first item, 1 after the current first etc.

Reply:
Updated; please check again when you have a chance. Thanks